libs/ui/elements/src/lib/related-record-card/related-record-card.component.html

```diff
@@ -1,27 +1,28 @@
 <div
   class="w-72 h-96 overflow-hidden rounded-lg shadow-primary-light bg-white cursor-pointer"
-  [routerLink]="['/dataset', record.uuid]"
 >
-  <div class="h-52 bg-gray-100">
-    <gn-ui-thumbnail
-      class="h-52 w-full object-cover"
-      [thumbnailUrl]="record.thumbnailUrl"
-    ></gn-ui-thumbnail>
-  </div>
-  <div class="flex flex-col justify-between h-44 px-5 pt-4 pb-6">
-    <h4
-      class="max-h-24 font-title text-21 text-black overflow-ellipsis overflow-hidden"
-    >
-      {{ record.title }}
-    </h4>
-    <div>
-      <button
-        mat-raised-button
-        [matTooltip]="'tooltip.url.open' | translate"
-        matTooltipPosition="above"
+  <a [routerLink]="['/dataset', record.uuid]" target="_blank">
```

Comment:
why not make the encompassing `div` an `a` instead? with `display-block` it should give the same result I think

Reply:
Indeed, thanks for the hint!